loadpipeofficial/Hamza-medusa
# Ham-1742: reviews homepage

- ProductCardGroup is now using ssr component
- Hydration added to account/review page
- Create a review still does not update the homepage reviews....

TO TEST;
- Go to homepage
- Apply filters
- Go to Account/Reviews 
- Check if your reviews are showing

This task is `incomplete`: https://www.notion.so/hamza-market-token/Intermittent-Reviews-not-displaying-in-Home-Page-09691dfc04174136a96c5d5b926b9b52?pvs=4
